django_dbq/models.py

```diff
@@ -58,7 +58,9 @@ class Meta:
     objects = JobManager()
 
     def save(self, *args, **kwargs):
-        if not self.pk:
+        is_new = not Job.objects.filter(pk=self.pk).exists()
```

Comment:
Then perhaps this initialisation step should be in a model classmethod or manager method for creating the job?